package.json

```diff
@@ -51,7 +51,7 @@
     "mocha": "^8.4.0",
     "mocha-headless-chrome": "^3.1.0",
     "nyc": "^15.0.1",
-    "orbit-db-benchmark-runner": "^1.0.3",
+    "orbit-db-benchmark-runner": "github:tabcat/benchmark-runner#benchmarker",
```

Comment:
I suppose it's ok to publish a new version of the bechmark runner at this point?

Reply:
I published `orbit-db-benchmark-runner@2.0.0` so you can use that here.